src/SAHB.GraphQL.Client.Subscription/GraphQLSubscriptionWebSocketClient.cs

```diff
@@ -31,6 +31,17 @@ public GraphQLSubscriptionWebSocketClient(IGraphQLFieldBuilder fieldBuilder, IGr
             this.deserialization = deserialization;
         }
 
+        public GraphQLSubscriptionWebSocketClient(ClientWebSocket websocket, IGraphQLFieldBuilder fieldBuilder, IGraphQLQueryGeneratorFromFields queryGenerator, IGraphQLDeserialization deserialization, CancellationToken cancellationToken)
+        {
+            _webSocket = websocket;
```

Comment:
This variable can now be instantiated twice, since line 15 instantiates the websocket to new ClientWebSocket().
Please change line 15 to:
`private readonly ClientWebSocket _webSocket;`

Reply:
Ah, I see - I been trying to come at it from the other direction. I'll give it a check over and try to commit an improvement.
Should I then do another pull request, or is there a way to 'update' this one? (I use subversion more than git and am still getting to grips with it!)

Reply:
Should I be doing this change on master or develop...? I've just done a fresh checkout of my master branch, but get a whole load of 'type/namespace not found' errors in the examples... :S
e.g. 
using SAHB.GraphQLClient.Deserialization;
using SAHB.GraphQLClient.FieldBuilder.Attributes;
using SAHB.GraphQLClient.Executor;
using SAHB.GraphQLClient.Extentions;
using SAHB.GraphQLClient.QueryGenerator;
are all not found.
No errors with the develop branch. (I cheated a bit yesterday and tried the mod out on the develop branch as that was the one I'd been working with).
Is there a reference or something missing?

Reply:
In general you should base your PR on the develop branch. I'm using a branching strategy called [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/).
If it's a hotfix I will include this in a hotfix release.